### PR TITLE
More documentation for keras adagrad and adadelta

### PR DESCRIPTION
**Pull request by maxpumperla**
Replicating PR https://github.com/keras-team/keras/pull/10410 from `keras`, as suggested by @fchollet

### CODE REVIEW

File: tensorflow/python/keras/optimizers.py [hunk 1]
```diff
@@ -314,17 +314,24 @@ def get_config(self):
 
 @tf_export('keras.optimizers.Adagrad')
 class Adagrad(Optimizer):
-  """Adagrad optimizer.
+    """Adagrad optimizer.
 
-  It is recommended to leave the parameters of this optimizer
-  at their default values.
+    Adagrad is an optimizer with parameter-specific learning rates,
+    which are adapted relative to how frequently a parameter gets
```
[the remark this answers is not in the document]
Comment (maxpumperla) [reply tensorflow/python/keras/optimizers.py]: @caisq are you sure? this is a rather common formulation and I do mean an adaption _relative to x_ (not _adapt_ relatively, but _is_ relative), as e.g. used here:

https://books.google.es/books?id=omIpBgAAQBAJ&pg=PA58&lpg=PA58&dq=%22is+adapted+relative%22&source=bl&ots=q8_AXXgfnX&sig=vxktLNEYCCotiMMREPza-evZn-s&hl=en&sa=X&ved=0ahUKEwidlL_C19zbAhUJSsAKHZZwDsMQ6AEINDAC#v=onepage&q=%22is%20adapted%20relative%22&f=false

**Comment by tensorflowbutler** [comment]
Nagging Reviewer @fchollet: It has been 18 days with no activity and the `awaiting review` label was assigned. Can you please take a look?

File: tensorflow/python/keras/optimizers.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (caisq) [reply tensorflow/python/keras/optimizers.py]: @maxpumperla On second look, you are right. The usage of `relative to` here looks correct to me.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @caisq: It has been 46 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Review by caisq — DISMISSED** [review]

**Comment by maxpumperla** [comment]
cool. can we merge this then?

**Review by fchollet — CHANGES_REQUESTED** [review]
Looks good, but please use an indentation of 2 (you're using 4).

**Comment by tensorflowbutler** [comment]
Nagging Assignee @caisq: It has been 20 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @caisq: It has been 35 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by maxpumperla** [comment]
@fchollet ok, fixed. good to go.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @caisq: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.
